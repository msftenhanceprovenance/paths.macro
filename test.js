const babel = require('@babel/core');

const options = {
  plugins: [
    require.resolve('babel-plugin-macros'),
  ],
};

test('should transform', () => {
  const inputFile = './src/input.js';

  const result = babel.transformFileSync(inputFile, options);

  expect(result.code).toMatchSnapshot();
});
