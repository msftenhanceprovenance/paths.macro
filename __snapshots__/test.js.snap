// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should transform 1`] = `
"console.log(\\"/src/\\", \\"input\\");
alert(\\"input\\", \\"/src/\\");

function usePathsForSomething() {
  return [\\"input\\", \\"/src/\\"];
}"
`;
